tests/rptest/services/redpanda.py

```diff
@@ -574,14 +576,34 @@ def start_node(self,
         if self.coproc_enabled():
             self.start_wasm_engine(node)
 
+        def is_status_ready():
+            status = None
+            try:
+                status = Admin.ready(node).get("status")
+            except:
+                self.logger.error(
+                    f"error on getting status from {node.account.hostname}")
+                e, v = sys.exc_info()[:2]
+                trace = traceback.format_exc()
+                self.logger.error(e)
+                self.logger.error(v)
+                self.logger.error(trace)
+                raise
+            if status != "ready":
+                self.logger.debug(
+                    f"status of {node.account.hostname} isn't ready: {status}")
+                return False
+            return True
```

Comment:
The `Admin.ready(node).get("status")` is low-level and predates the improved `Admin._requests` wrapper, so the extra debug logging isn't surprising to see.

The `Admin.get_status_ready` interface uses the `Admion._request` wrapper which adds retry logic and better logging etc... Since the logging here in the exception handler looks like it is generic to the endpoint specific ready endpoint, it may be better to add this extra logging that you've found useful into that `Admin._request` wrapper.

Reply:
The `ready` method is a little bit special in that it's static, and it should never be retried (if a readiness probe call fails, that's your answer to "is it ready").

Reply:
Seems like `Admin._request` should take a no-retry flag rather than building a one-off wrapper that only adds logging.